### DISCUSSION TITLE
Build errors using grapesjs-plugin-forms:2.0.5 with grapesjs:0.21.1

### DISCUSSION THREAD

**Discussion by codalf** [GrapesJS/grapesjs#5145] [Q&A]
Hi,

I was trying to use the latest version of grapesjs-plugin-forms together with grapesjs 0.21.1
I am getting the following build errors in my project:

```
node_modules/grapesjs-plugin-forms/dist/index.d.ts:13:13 - error TS2503: Cannot find namespace 'grapesjs'.

13  category?: grapesjs.BlockOptions["category"];
               ~~~~~~~~

node_modules/grapesjs-plugin-forms/dist/index.d.ts:21:23 - error TS2503: Cannot find namespace 'grapesjs'.

21 declare const plugin: grapesjs.Plugin<PluginOptions>;
```

My question: Does that mean, that these versions are not compatible? I also wasn't able to get an older version of the plugin to run with grapesjs 0.21.1
Any idea, what needs to be fixed to get this working?

Thanks,
Peter

**Comment by artf** [comment]
Yeah, we need to update the TS for some of the plugins,  PRs are welcome.
